blog/_src/posts/2017-05-12-artifacts-for-semantics.md

```diff
@@ -0,0 +1,155 @@
+    Title: Artifacts for Semantics
+    Date: 2017-05-12T10:08:31
+    Tags: by Daniel Patterson
+
+[Gabriel Scherer](http://gallium.inria.fr/~scherer/) and I recently wrote
+an [artifact](https://dbp.io/artifacts/funtal) for a
+semantics [paper](https://dbp.io/pubs/2017/funtal.pdf) on a typed assembly
+language interoperating with a high-level functional language. Building
```

Comment:
How about
- adding a line break before `Building` and
- putting the artifact URL on its own line?